ee/modules/500-operator-trivy/templates/configmap.yaml

```diff
@@ -60,3 +60,4 @@ data:
   TRIVY_DEBUG: "false"
   TRIVY_SKIP_DB_UPDATE: "false"
   TRIVY_DB_REPOSITORY: {{ $dbRepository }}
+  TRIVY_REGISTRY_CA: | {{ $.Values.global.modulesImages.registry.CA | nindent 4 }}
```

Comment:
Didn't get this. What for do we need this?

Reply:
In addition to the scan job, which needs CA to download the scanned image, trivy server also needs CA to download the vulnerability database